llvm/test/Transforms/MemCpyOpt/memcpy-zero-size.ll

```diff
@@ -34,3 +34,18 @@ define void @pr64886(i64 %len, ptr noalias %p) {
   call void @llvm.memcpy.p0.p0.i64(ptr inttoptr (i64 -1 to ptr), ptr %p, i64 poison, i1 false)
   ret void
 }
+
+define void @pr98610(ptr %p, ptr noalias %p2) {
```

Comment:
Hence, it is irrelevant which is the `noalias` pointer in the context of this issue, right?